Composer/packages/lib/code-editor/src/hooks/useSelection.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { ISelection, Selection, SelectionMode } from 'office-ui-fabric-react/lib/DetailsList';
+import * as React from 'react';
+
+export const useSelection = <T>(selectionMode?: SelectionMode) => {
```

Comment:
We should put this hook in the ui-shared package so we can use it else where in the app.

Reply:
removed it